schema/spacewalk/upgrade/susemanager-schema-4.0.9-to-susemanager-schema-4.0.10/150-drop_state_change.sql.postgresql

```diff
@@ -0,0 +1,19 @@
+-- oracle equivalent source sha1 26459fcfa8133dcbc7745e7282246a18de66a509
+--
+--
+-- Copyright (c) 2019 SUSE LLC
+--
+-- This software is licensed to you under the GNU General Public License,
+-- version 2 (GPLv2). There is NO WARRANTY for this software, express or
+-- implied, including the implied warranties of MERCHANTABILITY or FITNESS
+-- FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+-- along with this software; if not, see
+-- http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+--
+-- Red Hat trademarks are not licensed under GPLv2. No permission is
+-- granted to use or replicate Red Hat trademarks that are incorporated
+-- in this software or its documentation.
+--
+
+DROP TABLE IF EXISTS state_change;
```

Comment:
I think you need to drop the index as well? Or is it dropped automatically when the table is gone?

Reply:
On postgres it works fine. But I'll check on oracle, just to be sure.

Reply:
Oracle? We don't have Oracle anymore :)

Reply:
Ok, oracle is also safe.

Reply:
Ah, I see - all the migration files are empty now (last time I added the scripts it wasn't the case :smile:  ). Even better.

Reply:
And the index is really gone on postgresql? I know that you can drop tables and indexes sometimes kept.